bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Tally/TallyMediator.cs

```diff
@@ -0,0 +1,38 @@
+namespace ElectionGuard.Decryption.Tally;
+
+public class TallyMediator : DisposableBase
```

Comment:
i went back and forth on wether or not to even make this class but it's here. it's basically just a stub for when we hook it up to the database